src/main/kotlin/uk/gov/justice/hmpps/offenderevents/services/HMPPSDomainEventsEmitter.kt

```diff
@@ -455,6 +458,7 @@ class HMPPSDomainEventsEmitter(
         .withAdditionalInformation("bookingId", bookingId)
         .withAdditionalInformation("movedToNomsNumber", offenderIdDisplay)
         .withAdditionalInformation("movedFromNomsNumber", previousOffenderIdDisplay)
+        .withAdditionalInformation("bookingStartDateTime", bookingStartDateTime)
```

Comment:
only interested in the bookingStartDate so didn't bother to add in lastAdmissionDate